website/docs/routes/social-media.mdx

```diff
@@ -1704,6 +1704,12 @@ rule
 
 <Route author="xyqfer" example="/weibo/search/hot" path="/weibo/search/hot" anticrawler="1" radar="1" rssbud="1"/>
 
+### 热搜榜(含摘要) {#wei-bo-re-sou-bang-content}
+
+<Route author="shinemoon" example="/weibo/search/hot/fulltext" path="/weibo/search/hot/fulltext" anticrawler="1" radar="1" rssbud="1"/>
```

Comment:
Please merge with the current docs above instead of creating a new one.

Reply:
I tried to merge but not so sure about format/expectation on the doc side, pls help to double check and let me know if something not ideal there. thx!